lib/serial_terminal.pm

```diff
@@ -96,4 +101,105 @@ sub serial_term_prompt {
     return $serial_term_prompt;
 }
 
+
+=head2 download_file
+
+  download_file($src, $dst [, force => $force][, chunk_size => $cz][, chunk_retry => $cr])
+
+Download a file from worker to SUT using the current serial terminal.
+The file is parted into chunks C<chunk_size> and each chunk gets is verified with
+checksum. If a chunk fail we retry it C<chunk_retry> times, before give up.
+To override destination us C<force>.
+This function die on any failure.
+=cut
+sub download_file {
+    my ($src, $dst, %opts) = @_;
+    $opts{chunk_size}  //= 1024 * 2;
+    $opts{chunk_retry} //= 16;
+    $opts{force}       //= 0;
+    record_info('Download file', "From worker ($src) to SUT ($dst)");
```

Comment:
instead of SUT, I would say to VM as this is a generic place, not wicked related.

Reply:
For me, SUT is quite generic. If we call it VM, what  about baremetal?

Reply:
Right, VM not valid for Bare-metal.. but SUT just reminds me a lot of Wicked SUT/REF. So, if we do this in REF VM is not SUT... But ok to keep it like this.

Reply:
Ah, now I get it. That's true, we are using a SUT (from openqa perspective) as REF in wicked.
We could call it host.